pyqtorch/circuit.py

```diff
@@ -122,6 +122,84 @@ def flatten(self) -> ModuleList:
         return ModuleList(ops)
 
 
+class DropoutQuantumCircuit(QuantumCircuit):
+    def __init__(
+        self,
+        n_qubits: int,
+        operations: list[Module],
+        dropout_mode: DropoutMode = DropoutMode.ROTATIONAL,
+        dropout_prob: float = 1.0,
+    ):
+        super().__init__(n_qubits, operations)
+        self.dropout_mode = dropout_mode
+        self.dropout_prob = dropout_prob
+
+        self.dropout_fn = getattr(self, dropout_mode)
+
+    def forward(self, state: State, values: dict[str, Tensor] | ParameterDict = {}) -> State:
```

Comment:
we dont need the if...elif. we are always using dropout now right?

Reply:
kept in case you want to use the circuit at test time and do not want dropout present